obj/rider.project.restore.info

```diff
@@ -0,0 +1 @@
+﻿16521848118827735
```

Comment:
Please remove the obj directory from the project, its not supposed to be in the repository since it contains generated data (also specific to your PC as can be seen in the source of the files)

Reply:
Removed